EKrut-Client/src/ekrut/client/managers/ClientSessionManager.java

```diff
@@ -92,14 +95,21 @@ public boolean isLoggedin() {
 		return false;
 	}
 
+	//if someone need to fetchUser by somthing, he need to call to fetchUser() function 
+	//and then call to getUserList() or getUser()
+	public void fetchUser(FetchUserType fetchType, String argument) {
+		UserRequest userRequest = new UserRequest(fetchType,argument);
+		UserResponse userResponse = sendRequest(userRequest);
+		if(fetchType == FetchUserType.ROLE) 
+			usersList =  userResponse.getUsersList();
+		user = userResponse.getUser();
```

Comment:
`user` is the currently logged in user, you can't do something like that.

Reply:
You still have not fixed the issue. You're overwriting the logged in user with a random user that someone fetched.